test/integration/verify/controls/aws_cloudwatch_log_metric_filter.rb

```diff
@@ -0,0 +1,46 @@
+lmf_1_name = attribute(
+  'lmf_1_name',
+  default: 'default.lmf_1_name',
+  description: 'Name of a Cloudwatch Log Metric Filter',
+)
+
+lmf_2_name = attribute(
+  'lmf_2_name',
+  default: 'default.lmf_2_name',
+  description: 'Name of a Cloudwatch Log Metric Filter',
+)
+
+lmf_lg_1_name = attribute(
+  'lmf_lg_1_name',
+  default: 'default.lmf_lg_1_name',
+  description: 'Name of a Cloudwatch Log Group',
+)
+
+lmf_lg_2_name = attribute(
+  'lmf_lg_2_name',
+  default: 'default.lmf_lg_2_name',
+  description: 'Name of a Cloudwatch Log Group',
+)
+
+lmf_1_metric_1_name = attribute(
+  'lmf_1_metric_1_name',
+  default: 'default.lmf_1_metric_1_name',
+  description: 'Name of a Cloudwatch Metric',
+)
+
+describe aws_cloudwatch_log_metric_filter(
+  filter_name: lmf_1_name,
+  log_group_name: lmf_lg_1_name,
+) do
+  it { should exist }
+  its('pattern') { should cmp 'testpattern01'}
```

Comment:
I see we use `cmp` here but `be` in the examples. Can we standardize?

Reply:
Good point.  I think @adamleff has said in the past we'd like to encourage the use of `cmp` for its flexibility and "do-what-I-mean"-ness.  I'll use cmp throughout.